Dont use the qt+libs system for managing Qt dependencies, because Nix provides
them already.
Don't build the deploy.lisp helper file, because Nix also can handle deployment.
--- a/qtools.asd
+++ b/qtools.asd
@@ -33,10 +33,9 @@
                (:file "generate")
                (:file "dynamic")
                (:file "precompile")
-               (:file "deploy")
                (:file "fast-call")
                (:file "documentation"))
-  :depends-on (:qt+libs
+  :depends-on (:qt
                :deploy
                :cl-ppcre
                :closer-mop

Diff finished.  Sun Oct  2 14:38:06 2022